src/MICore/Debugger.cs

```diff
@@ -569,6 +571,13 @@ public Task CmdBreakInternal()
                     return CmdUnixBreak(debuggeePid, ResultClass.done);
                 }
             }
+            else if (_transport is PipeTransport)
```

Comment:
Yes, that would be a better check. I'll also check for gdb since this is only a gdb scenario (at least for now).